.gitlab-ci.yml

```diff
@@ -41,6 +41,31 @@ build:docker:
     - when: never
   script: ["true"]
 
+build:archive:
+  rules:
+    - when: manual
+      allow_failure: true
+  image: "golang:${GO_VERSION}"
+  stage: build
+  script:
+    - export VERSION=$(git describe --tags --dirty)
+    - export CGO_ENABLED=0
+    - |-
+      echo -n "${MULTIPLATFORM_PLATFORMS}" | \
+          tr ',' '\n' | tr '/' ' ' | \
+          while read -r GOOS GOARCH VARIANT; do
+              export GOOS GOARCH;
+              if test "${GOARCH}" = "arm"; then
+                  export GOARM="${VARIANT#v}"
+              fi
+              export DESTDIR="dist/nt-connect_${VERSION}_${GOOS}_${GOARCH}${VARIANT}"
+              make install
+              tar -C "${DESTDIR}" -cJf "${DESTDIR}.tar.xz" .
```

Comment:
Consider moving this into a make target: `make dist` could do the install into DESTDIR and get you the tarball.